openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+{% spaceless %}
+
+    {% block body %}
+    <div class="staff__grade__form ui-toggle-visibility__content" data-submission-uuid="{{ submission.uuid }}">
```

Comment:
@cptvitamin This is the form that replaces the div where I commented above with class "staff__grade__form" (after the user clicks).

The other thing that seems confusing for accessibility is that when you click the action buttons below ("Submit Assessment", and "Submit Assessment and Grade Another Learner"), the HTML re-renders after the assessment is submitted to the server. I'm not sure if focus goes to an appropriate place to make it clear what happened.